AirCasting/SessionViews/MeasurementDotView.swift

```diff
@@ -2,30 +2,47 @@
 //
 
 import SwiftUI
+import Resolver
 
 struct MeasurementDotView: View {
     
     let value: Double
     @ObservedObject var thresholds: SensorThreshold
-    
+    @InjectedObject private var userSettings: UserSettings
+
     var body: some View {
         color
             .clipShape(Circle())
             .frame(width: 5, height: 5)
     }
     
     var color: Color {
-        switch Int32(value) {
-        case thresholds.thresholdVeryLow ..< thresholds.thresholdLow:
-            return Color.aircastingGreen
-        case thresholds.thresholdLow ..< thresholds.thresholdMedium:
-            return Color.aircastingYellow
-        case thresholds.thresholdMedium ..< thresholds.thresholdHigh :
-            return Color.aircastingOrange
-        case thresholds.thresholdHigh ... thresholds.thresholdVeryHigh :
-            return Color.aircastingRed
-        default:
-            return Color.aircastingGray
+        if thresholds.sensorName == MeasurementStreamSensorName.f.rawValue && userSettings.convertToCelsius {
```

Comment:
I think you wrote a function which you can use it instead of duplicating the code.

```
var color: Color {
        if thresholds.sensorName == MeasurementStreamSensorName.f.rawValue && userSettings.convertToCelsius {
            return thresholds.colorForCelsius(value: Int32(value))
        } else {
            return thresholds.colorFor(value: Int32(value))
        }
    }
```